QualityControl/public/pages/aboutView/AboutViewModel.js

```diff
@@ -30,10 +30,10 @@ export default class FrameworkInfo extends Observable {
   }
 
   /**
-   * Load FrameworkInfo into `item`
+   * Load About Model into `item`
```

Comment:
The model is the class itself. 
This method does not `Load About Model into `item``. 
In item, data about the server of the GUI and associated services is stored in a remotedata. A remote data is used so that a nice UX is provided to the User, to know when items are loading, failed, or successfully loaded.